src/main/java/me/senseiwells/essentialclient/mixins/clientScript/StonecutterScreenHandlerMixin.java

```diff
@@ -0,0 +1,34 @@
+package me.senseiwells.essentialclient.mixins.clientScript;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.recipe.StonecuttingRecipe;
+import net.minecraft.screen.Property;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.List;
+
+import static me.senseiwells.essentialclient.clientscript.events.MinecraftScriptEvents.ON_CLICK_STONECUTTER_RECIPE;
```

Comment:
There's no need for a static import here, all other events just access the static field.